### ISSUE TITLE
How to check whether tests are failed or not on running newman tests with command line commands

### ISSUE THREAD

**Issue by abhishekprsetty** [postmanlabs/newman#1535]
1. Newman Version (can be found via `newman -v`): 3.9.3
 2. OS details (type, version, and architecture): Ubuntu 16
 3. Are you using Newman as a library, or via the CLI? Yes
 3. Did you encounter this recently, or has this bug always been there:NA
 4. Expected behaviour:
If I am running following command, How to check if the tests are failed, execute some number of tests. Could you please help how I can check with newman if the tests are failed or not apart from GUI.

 5. Command / script used to run Newman:

def runPostmanTests(){
    def SSH_CMD="newman run  /home/ingestion.postman_collection.json -e /home/env.json --bail newman --reporters cli,json --reporter-json-export outputfile.json"
    sh "$SSH_CMD"
}

**Comment by prakharjoshi** [comment]
@abhishekprsetty Can you share the sample collection that you are executing with Newman. You are bailing out if there is any error and skipping the Newman folder. Can you confirm, it is not the case? Also, If everything works fine then the tests result will be available at the `outputfile.json` which will be created in the same folder from where you are running the Newman.

**Comment by abhishekprsetty** [comment]
Yes, Its correct. /ingestion.postman_collection.json this is just collection of apis to run where I am checking only for response status code. 

 Could you please help how I can check with newman if the tests are passed or failed by executing the command.?

**Comment by kunagpal** [comment]
@abhishekprsetty The command will exit with a status code of 1 if any tests have failed, or other errors have occurred. You can use `$?` variable immediately after the newman run command in your snippet to check for the state of completed runs. Alternatively, you can add `set -e;` at the very beginning of your script, so that any failed command automatically causes the script to bail out with a non-zero exit code.
